### ISSUE TITLE
Conky

### ISSUE THREAD

**Issue by vlp-git** [madhur/dotfiles#4]
Hello,

How do you manage you conky windows on differents workspaces ? That would be better to stick it in ws1 for exemple.

Because, in a ws with you browser, conky is in front of the borwser window and it is pretty unusable.

Thanks

**Comment by madhur** [comment]
yeah, I have this hotkey to toggle the gaps, whenever I want to look at conky

https://github.com/madhur/dotfiles/blob/main/home/madhur/.config/sxhkd/config_i3#L21

```
super + m
	i3-msg gaps horizontal current toggle 400
```

Pressing this gives 400px of gaps on both side of window, allowing me to peek at conky.

**Comment by madhur** [comment]
As a general principle, if I want to monitor something more often, I rather put it in the i3 bar. I use conky for less often use cases.

**Comment by vlp-git** [comment]
ohhh thanks for the tips !! Why do you use sxhkd instead of sway/i3 shortcut ?

**Comment by madhur** [comment]
Few reasons:
* Like to keep keybindings separate. I have a dedicated sxhkd config for wm independent key  bindings
https://github.com/madhur/dotfiles/blob/main/home/madhur/.config/sxhkd/sxhkdrc

This makes it easier to try different window managers by keeping some standard key bindings common and avoid re configuring in every WM.

* There is a script which can read from sxhkd config and show them as rofi menu
https://github.com/madhur/dotfiles/blob/main/home/madhur/scripts/sxhkd-cheat

Though there is a downside that sometimes i3 starts but not sxhkd, leaving me blind without any shortcuts. 

For this there is jgmenu configured, which triggers on right click of i3 bar to spawn common applications such as conky, picom, sxhkd
https://github.com/madhur/dotfiles/blob/main/home/madhur/.config/jgmenu/custom.csv
